clients/imodels-client-management/src/base/types/RestClient.ts

```diff
@@ -43,6 +43,20 @@ export interface HttpRequestParams {
   headers: Dictionary<string>;
 }
 
+/** Abstraction for accessing HTTP response headers that the server responded with. */
+export interface HttpResponseHeaders {
+  /** Gets HTTP response header's value by the specified {@link headerName}. */
+  get(headerName: string): unknown;
```

Comment:
I think we should keep an option to just get all headers hat came with response, in particular for debugging and maybe future logging purposes. maybe an additional metod, like `getAll(): ...`?

Reply:
added